source/mobile/consider-mobile-vpn-options.rst

```diff
@@ -0,0 +1,52 @@
+Consider Mobile VPN Options
+===========================
+
+Connecting to your Private Network Mattermost Instance
+------------------------------------------------------
+
+You need to set up a way to connect to your private network Mattermost instance, using an external proxy with encrypted transport through HTTPS and WSS network connections.
+
+Depending on your security policies, we recommend deploying Mattermost behind a VPN and using a `per-app VPN <#id3>`_ with your EMM provider, or a mobile VPN client.
+
+Also consider deploying a mobile VPN client with multi-factor authentication (MFA) to your preferred login method, such as GitLab SSO with MFA, or run Mattermost Enterprise Edition with `multi-factor authentication (MFA) <https://docs.mattermost.com/deployment/auth.html>`__ enabled.
+
+Consider Mobile VPN Options
```

Comment:
this subtitle is the same as the page title, is that ok?

Reply:
Good call -- I'm going to update that slightly. Love your reviews, @enahum.